src/routes/api/voting.js

```diff
@@ -15,87 +15,92 @@ module.exports = function(express) {
  * This sets the util voting to equal the path to the lib/util
 */
 
-  /** Read One. */
-  router.get('/voting/:vote_id', function(req, res) {
 /**
  * @function get
  * @description Gathers Data
- * @param req 
+ * @param req
  * Requests Data
  * @param res
- * Listens for response  
+ * Listens for response
```

Comment:
This is what the QA is for I also have updated my files I am not looking forward to the re base and merging of these files if you see issues please QA next time thanks. #22